src/ex1/index.html

```diff
@@ -3,12 +3,29 @@
 <head>
```

Comment:
in general you used the name of the task as the identifier for deletion and stuff, 
that enforced you to have the limitation of two tasks with the same name (which you handled well), and even if this is your desired behavior I won't do that so if someday you'll want to change that you won't have to change your logic